### ISSUE TITLE
test beancount.plugins.forecast_test.TestExampleForecast fails

### ISSUE THREAD

**Issue by blais** [beancount/beancount#266]
**[Original report](https://bitbucket.org/blais/beancount/issue/266) by Martin Michlmayr (Bitbucket: [tbm13](https://bitbucket.org/tbm13), GitHub: [tbm](https://github.com/tbm)).**

----------------------------------------

Test beancount.plugins.forecast_test.TestExampleForecast fails because of 2 issues:

1) It's time dependent.  Currently it creations forecasts until the end of the current year, so running it in different years will give different results

2) It seems previously it created a transaction only starting in the FOLLOWING month whereas now it starts with the current month (which imho makes more sense).

I'll open a PR with a fix.

```
FAIL: test_forecast (beancount.plugins.forecast_test.TestExampleForecast)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/tbm/beancount/beancount/plugins/forecast_test.py", line 59, in test_forecast
    """, entries)
  File "/home/tbm/beancount/beancount/parser/cmptest.py", line 86, in assertEqualEntries
    self.fail(oss.getvalue())
AssertionError: Present in actual set and not in expected set:

....
LOTS OF LINES REMOVED
...

2018-11-17 # "Something"
  Expenses:Restaurant   50.02 USD
  Assets:Cash          -50.02 USD

2018-12-17 # "Something"
  Expenses:Restaurant   50.02 USD
  Assets:Cash          -50.02 USD



----------------------------------------------------------------------
Ran 1 test in 0.023s

FAILED (failures=1)
```

**Comment by blais** [comment]
**Original comment by Martin Blais (Bitbucket: [blais](https://bitbucket.org/blais), GitHub: [blais](https://github.com/blais)).**

----------------------------------------

Merged in tbm13/beancount/bug266 (pull request #67)

Fixed #266: ensure the test of the forecast plugin are not time dependent

Approved-by: Martin Blais <blais@furius.ca>

**Comment by blais** [comment]
**Original comment by Martin Blais (Bitbucket: [blais](https://bitbucket.org/blais), GitHub: [blais](https://github.com/blais)).**

----------------------------------------

Fixed #266: ensure the test of the forecast plugin are not time dependent

**Comment by blais** [comment]
**Original comment by Martin Blais (Bitbucket: [blais](https://bitbucket.org/blais), GitHub: [blais](https://github.com/blais)).**

----------------------------------------

Merged in tbm13/beancount/bug266 (pull request #67)

Fixed #266: ensure the test of the forecast plugin are not time dependent

Approved-by: Martin Blais <blais@furius.ca>
